lib/features/taskcluster_proxy.js

```diff
@@ -33,10 +33,16 @@ export default class TaskclusterProxy {
 
     var cmd = [
         '--client-id=' + task.runtime.taskcluster.clientId,
-        '--access-token=' + task.runtime.taskcluster.accessToken,
+        '--access-token=' + task.runtime.taskcluster.accessToken
+    ];
+    // only pass in a certificate if one has been set
+    if (task.runtime.taskcluster.certificate && task.runtime.taskcluster.certificate.length > 0) {
```

Comment:
I don't think you need ot check for length because if the string is empty will fail the check for it being defined.